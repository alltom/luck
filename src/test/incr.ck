<<< ++(int a), "should be 1" >>>;
<<< a++, "should be 1" >>>;
<<< a--, "should be 2" >>>;
<<< --a, "should be 0" >>>;
