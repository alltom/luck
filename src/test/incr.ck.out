1 should be 1 
1 should be 1 
2 should be 2 
0 should be 0 
